src/app/modules/auth/auth.utils.ts

```diff
@@ -56,3 +57,7 @@ export const mapRouteError = (
       }
   }
 }
+
+export const isUserInSession = (req: Request): boolean => {
+  return !!req.session?.user
+}
```

Comment:
at some point we should also save the user ID and the user object into session, so that we can migrate off the user object at our convenience.